Automattic/wp-calypso
# CI: clean up Chromedriver and references from repo.

#### Changes proposed in this Pull Request

This PR removes dependencies and references to Chromedriver from the repository.

Key changes:
- removal of references to `CHROMEDRIVER_SKIP_DOWNLOAD` and `CHROMEDRIVER_SKIP_DOWNLOAD` env vars;
- removal of dependency on `chromedriver@88.x.x` from yarn;
- discontinue installation of Chrome in the docker image.

We no longer use Selenium in this repository as evidenced by https://github.com/Automattic/wp-calypso/pull/63108. Since Playwright manages its own browser installations, it is safe to remove Chrome and Chromedriver from the generated docker image.

#### Testing instructions

Ensure the following:
  - [x] P2 E2E

Related to https://github.com/Automattic/wp-calypso/pull/63108
